tree/dataframe/inc/ROOT/RDF/InterfaceUtils.hxx

```diff
@@ -44,6 +44,7 @@
 class TObjArray;
```

Comment:
`InterfaceUtils.hxx` is a header for utilities `RInterface` depends on, not for utilities that depend on `RInterface`. I'd rather avoid the circular logical dependency.

Features like this one could go in `RDFHelpers.hxx` instead (I'm open to better suggestions).

I just realized that `TriggerRun` does not belong in this header either, for the same reason, but that's for another PR.